crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1252,12 +1252,13 @@ fn get_bank_debit_data(
     }
 }
 
-fn create_stripe_payment_method(
+fn create_stripe_payment_method<F, Req, Res>(
     payment_method_data: &domain::PaymentMethodData,
     auth_type: enums::AuthenticationType,
     payment_method_token: Option<types::PaymentMethodToken>,
     is_customer_initiated_mandate_payment: Option<bool>,
     billing_address: StripeBillingAddress,
```

Comment:
you already have the billing address here right, do we need to use routerdata again here?